src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -58,6 +59,24 @@ public static void GetDirectoryName_SpaceOrControlCharsThrowOnWindows(string pat
             }
         }
 
+        [Theory,
+            InlineData(" "),
+            InlineData("\r\n")]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
+        public static void GetDirectoryName_SpaceOrControlCharsThrowOnWindows_Core(string path)
+        {
+            Action action = () => Path.GetDirectoryName(path);
+            if (PlatformDetection.IsWindows && path.Equals(" "))
```

Comment:
@JeremyKuhne should we be removing PathInternal.IsEffectivelyEmpty by the same reasoning as CheckInvalidPathChars?